code/API_definitions/location-verification.yaml

```diff
@@ -162,6 +162,8 @@ paths:
           $ref: "#/components/responses/Generic403"
         "404":
           $ref: "#/components/responses/Generic404"
+        "422":
+          $ref: "#/components/responses/RetrieveLocationUnprocessableEntity422"
```

Comment:
The current YAML file is location-verification.yaml, should this be VerifyLocationUnprocessableEntity422?

Reply:
Good catch!
Thanks @mhfoo !